src/components/containers/header/Header.js

```diff
@@ -78,9 +78,9 @@ class Header extends Component {
     }
 
     clearAuctionDetail(event) {
```

Comment:
I fear this was a feature I started working on and then forgot about.  Will anyone be using the add auction form on game day or is that only controlled by you before the auction?